AwsCryptographicMaterialProviders/runtimes/python/src/aws_cryptographic_materialproviders/smithygenerated/aws_cryptography_keystore/references.py

```diff
@@ -0,0 +1 @@
+# Code generated by smithy-python-codegen DO NOT EDIT.
```

Comment:
Remove if not required.

Reply:
Done, had to move some stuff around in Polymorph